content/docs/accessibility.md

```diff
@@ -46,18 +46,28 @@ in our websites will often give us accessibility for free.
 - [MDN HTML elements reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)
 
 Sometimes we break HTML semantics when we add `<div>` elements to our JSX to make our React code work, especially when working with lists (`<ol>`, `<ul>` and `<dl>`) and the HTML `<table>`.
-In these cases we should rather use React Fragments to group together multiple elements.
+In these cases we should rather use [React Fragments](/docs/fragments.html) to group together multiple elements.
 
-Use `<Fragment>` when a `key` prop is required:
- 
-```javascript{1,8,11}
+For example,
+
+```javascript{1,5,8,18,21}
 import React, { Fragment } from 'react';
 
+function ListItem({ item }) {
+  return (
+    <Fragment>
+      <dt>{item.term}</dt>
+      <dd>{item.description}</dd>
+    </Fragment>
+  );
+}
+
 function Glossary(props) {
   return (
     <dl>
       {props.items.map(item => (
-        // Without the `key`, React will fire a key warning
+        // When mapping a collection, the `key` prop is required.
+        // Without it, React will fire a key warning
```

Comment:
This duplicates information available elsewhere. I think it could be more terse:

Something like: "Fragments also require a `key` prop when mapping collections.

Reply:
Now that you mentioned it, I think it's not specifically Fragments that require a `key` prop but any component or element that is produced from mapping a collection.

Do I understand this right?

If so, what do you think of "Just like other components Fragments should have a `key` prop when mapping a collection"?

Reply:
Yes, there is indeed a `key` prop required on anything that is mapped for the React Reconciler to know when the collection change and not accidentally mess up the displayed sequence or omit elements.

You could say something like that, although the key is required on more than components. Also JSX elements, should they be mapped.

Reply:
I abstained from "anything" after I checked that mapping plain strings is okay without a `key` (well, there is nowhere one can put it either :) ). So it's anything that ends up being a `React.createElement()` result, right?

While we're polishing this sentence before I make the commit, which is acceptable?

1. "Just like other components and elements, Fragments should have a key prop when mapping a collection"
2. "Just like other JSX elements, Fragments should have a key prop when mapping a collection" (which doesn't cover render functions without JSX)
3. "Just like other React-elements, Fragments should have a key prop when mapping a collection"

?

As for the first part of these options, I feel that it would be better to make a connection in a reader's mind with the more common requirement of `key`, that they may or may not know of.

Reply:
I hate to be opinionated, LOL. but the best I can come up with is "Fragments also require a `key` prop when mapping collections" as that then does not need any clarification of what other elements required the key prop. Does not mean you have to use this verbatim, just something that does not get us in the mess of needing to explain something we explained ourselves into in the first place.

I also think that re-explaining keys is not required here as this is in the Advanced Guide section so we can assume that a the reader has already covered this basic concept.